jetty-alpn/jetty-alpn-openjdk8-server/pom.xml

```diff
@@ -12,7 +12,7 @@
   <name>Jetty :: ALPN :: OpenJDK8 Server Implementation</name>
 
   <properties>
-    <bundle-symbolic-name>${project.groupId}.alpn.conscrypt.server</bundle-symbolic-name>
+    <bundle-symbolic-name>${project.groupId}.alpn.openjdk8.server</bundle-symbolic-name>
```

Comment:
This looks good; however, shouldn't the client POM also changed in a similar way ?

Reply:
Done